### ISSUE TITLE
Loading Saved Models

### ISSUE THREAD

**Issue by sumitsinha** [cpmpercussion/keras-mdn-layer#12]
I was using the model for multi-dimensional prediction. Has some issues loading the model. What all custom objects and arguments need to be passed to load a saved Keras model.

**Comment by cpmpercussion** [comment]
I haven’t tested model saving/loading and it probably is not working. Saving and loading weights does work.

I’ll work on this a bit this week but if you come up with a solution or advice feel free to post here!

**Comment by cpmpercussion** [comment]
You'll just need to list the MDN layer and loss function in the custom_objects argument.

Unfortunately, you kind of need to know the parameters to generate the loss function, so the `load_model` method (where the configuration is fully encapsulated within the file) doesn't work that well. Here's an example for 0.2.3:

```
import mdn
import keras
N_HIDDEN = 5
N_MIXES = 5
model = keras.Sequential()
model.add(keras.layers.Dense(N_HIDDEN, batch_input_shape=(None, 1), activation='relu'))
model.add(mdn.MDN(1, N_MIXES))
model.compile(loss=mdn.get_mixture_loss_func(1, N_MIXES), optimizer=keras.optimizers.Adam())
model.save('test_save.h5')
m_2 = keras.models.load_model('test_save.h5', custom_objects={'MDN': mdn.MDN, 'mdn_loss_func': mdn.get_mixture_loss_func(1, N_MIXES)})
```
